lidar_localizer/test/test_launch_ndt_matching.test

```diff
@@ -1,10 +1,5 @@
 <launch>
-  <param name="tf_x" value="1.2" />
-  <param name="tf_y" value="0.0" />
-  <param name="tf_z" value="2.0" />
-  <param name="tf_yaw" value="0.0" />
-  <param name="tf_pitch" value="0.0" />
-  <param name="tf_roll" value="0.0" />
+  <rosparam param="tf_baselink2primarylidar">[1.2, 0.0, 2.0, 0.0, 0.0, 0.0]</rosparam>
```

Comment:
This significantly changes the API. While I agree that it is simpllier, we should leave this alone in this MR for backward-compatibility in this final feature release.

Reply:
Sounds good, I'll make it so the node tries TF first, then `tf_baselink2primarylidar` param, then finally the individual tf params.

Reply:
Since this is just the test launch file, I've left it as-is but the code will now try all three methods of getting the transforms.